_automate/bump_version.sh

```diff
@@ -10,9 +10,9 @@ ack "^version = \"" -l | \
 	grep toml | \
 	xargs sed -i "s/^version = \".*/version = \"$VERSION\"/"
 
-ack "{ version = \"$PREV_DEPS" -l | \
+ack "[{|\,] version = \"$PREV_DEPS" -l | \
 	grep toml | \
-	xargs sed -i "s/{ version = \"$PREV_DEPS/{ version = \"$NEW_DEPS/"
+	xargs sed -i "s/\([{|\,]\) version = \"$PREV_DEPS/\1 version = \"$NEW_DEPS/"
```

Comment:
Maybe better to unify syntax in the `test` crate?

Reply:
Why miss an opportunity to make an unreadable regex? :grin: